ogre/src/OgreCamera.cc

```diff
@@ -222,7 +222,8 @@ VisualPtr OgreCamera::VisualAt(const ignition::math::Vector2i
     }
   }
 
-  float ratio = screenScalingFactor();
+  // float ratio = screenScalingFactor();
+  float ratio = 1.0f;
```

Comment:
Should this be changed within `screenScalingFactor` as an alternative to #221 ?

Reply:
you mean update the `screenScalingFactor` to return just `1.0f`?

Reply:
Yeah... I mean, do we want anyone calling that function as it is now?

Reply:
like this b0fd1c7?

Reply:
Works for me :+1: